packages/core/src/browser/style/search-box.css

```diff
@@ -24,7 +24,7 @@
     box-shadow: var(--theia-border-width) var(--theia-border-width) var(--theia-widget-shadow);
     line-height: var(--theia-private-horizontal-tab-height);
     background-color: var(--theia-editor-background);
-    z-index: 1;
+    z-index: 1110;
```

Comment:
@vince-fugnitto @kaiyue0329 Maybe we can introduce css variable for toolbar z-index and here use calc? Otherwise it looks fragile.

Reply:
@akosyakov I agree, I think we should be smarter as not to potentially break in the future (fragile).